MiniAn/minian_main.py

```diff
@@ -183,19 +196,125 @@ def preview(minian_parameters):
     cluster.close()
 
 
+def penalties_preview(minian_params):
+    """
+    Saves penalties_preview
+    """
+
+    # We assume that initialization preview was generated first
+    # Read the saved data during the generation of the Initialization Preview
+    # It's in the "[TmpDir]/Intermediate", e.g. Y_hw_chk, Y_fm_chk
+
+    # Start cluster
+    print(mn_defs.Messages.START_CLUSTER, flush=True)
+    cluster = LocalCluster(**minian_params.params_LocalCluster)
+    annt_plugin = TaskAnnotation()
+    cluster.scheduler.add_plugin(annt_plugin)
+    client = Client(cluster)
+
+    intpath = os.path.join(minian_params.paths[defs.Parameters.Path.TMP_DIR], mn_defs.Folder.INTERMEDIATE)
+    subset = {"frame": slice(*minian_params.preview_params[defs.Parameters.Preview.RANGE])}
+
+    minian_ds  = open_minian(intpath)
+    Y_hw_chk = minian_ds["Y_hw_chk"]
+    Y_fm_chk = minian_ds["Y_fm_chk"]
+
+    print("Y_hw_chk", Y_hw_chk)
+    print("Y_fm_chk", Y_fm_chk)
+
+    file_, chk, _ = load_chunk(intpath, subset, minian_params)
+    seeds, _ = initialize_seeds(Y_fm_chk, Y_hw_chk, minian_params, True)
```

Comment:
Read seeds from minian preview hdf5 file

Reply:
Are we not missing some information? Because when we save to the hdf5 I think we don't save everything?